src/components/Navigation/index.tsx

```diff
@@ -46,22 +47,26 @@ const Navigation: React.FC<INavigation> = ({ readyToTransact, setSidebar }) => {
       <StyledNavigationItems>
         <ul>
           <li>
-            <StyledLink exact activeClassName="active" to="/propose">
-              Propose App
-            </StyledLink>
-          </li>
-          <li>
-            <StyledLink exact activeClassName="active" to="/new-farm">
-              New Farm
+            <StyledLink exact activeClassName="active" to="/">
+              <div>
```

Comment:
Hmm, I suppose the div is to force a `display: block` property for the SVGs—is there a way to do this (such as styling the link) without bloating the dom with unnecessary divs?

Reply:
Indeed there is a better way! Fixed in ccf4b95e644f999cb1f4831ab5e7cc7e9b715a4f